test/testautomation_audio.c

```diff
@@ -100,6 +100,12 @@ int audio_initQuitAudio()
 
         /* Call Init */
         result = SDL_AudioInit(audioDriver);
+#if defined(__linux__)
+        /* OSS is rarely seen on modern Linux distros, so failure is generally expected. */
+        if (result != 0 && SDL_strcmp(audioDriver, "dsp") == 0) {
```

Comment:
Are we ok with not allowing testing the dsp driver exclusively? e.g.
```
SDL_AUDIO_DRIVER=dsp test/testautomation
```

Reply:
Ah, sorry, that was supposed to be pushed to my own repo for testing.

But yes, we probably should explicitly allow it.  The tests which change the audio driver hint should save and restore the original hint so this is possible.